### ISSUE TITLE
accounts: Creating password with weak accounts uses a double-tap button, but should use an explicit override button instead

### ISSUE THREAD

**Issue by garrett** [cockpit-project/cockpit#16612]
When creating or changing the password, there's a weak password override. That's great.

However, it re-uses the same button. This is problematic as _everyone_ can accidentally click a second time and some people even have motor issues which cause them to press keys and click multiple times.

Current screenshot:

![Screenshot 2021-11-16 at 11-44-53 Accounts - garrett Bolt](https://user-images.githubusercontent.com/10246/141971415-76e50910-4e05-4163-b8c0-ac9de5b422b2.png)

**Comment by garrett** [comment]
### New create account password modal design

![create-account](https://user-images.githubusercontent.com/10246/141973961-9b2e4eaf-4bb8-4c4f-a154-dbcf89bccb67.png)

### With weak password error

![create-account-weak](https://user-images.githubusercontent.com/10246/141974017-6e5e641b-5e22-474e-9e08-c73820969bfa.png)

### With weak password error, but with the legacy design

![create-account-weak-legacy(1)](https://user-images.githubusercontent.com/10246/141974279-4afde41d-9b82-43c3-81c6-f4b25279019f.png)

### Source

- [create-account.excalidraw.zip](https://github.com/cockpit-project/cockpit/files/7545736/create-account.zip)
- https://excalidraw.com/#json=46P_Cp_fGDWB2sRk0N59D,OgvInW-3GJ51EGPdK_YoGg

---

### Notes

I've included the legacy design so you can focus on just making a minor change if you'd like. Otherwise, if you feel like overhauling the design in one go, the complete new design is above.

The change password needs similar adjustments. I can create mockups for that later, if you wish.

**Comment by garrett** [comment]
Design needs updating:

- Access should be Options
- Options should have Lock account as in the mockup
- Options should also have "Force password change on first use"
- Perhaps Options should also have "Require password change every [] days"?

**Comment by garrett** [comment]
But the design above works enough for this issue of having a confirmation button.

@jelle is creating additional separate issues for the other proposals.
